command/sub8_launch/config/buoy_finder.yaml

```diff
@@ -1,9 +1,10 @@
-camera_topic     : /camera/front/right/image_rect_color #Image topic to use to find buoys
-min_observations : 8 # Observations needed to attempt a least squares position estimate
-min_contour_area : 10 # Minimum pixels for a pottential buoy contour to not be rejected
-max_velocity     : 1 # Highest velocity sub can be moving as in m/s before observations are ignored
-min_trans        : 0.05 # Smallest movement of sub in meters between observations to count towards observation count
-timeout_seconds  : 2.5 # Time in which min_observations must be seen to be considered found
+camera_topic      : /camera/front/right/image_rect_color #Image topic to use to find buoys
+min_observations  : 8 # Observations needed to attempt a least squares position estimate
+min_contour_area  : 0.002 # Minimum of image area contour must be to not be rejected
+max_velocity      : 1 # Highest velocity sub can be moving as in m/s before observations are ignored
+max_circle_error  : 0.4 #max 0-1 error in contour area vs enclosing circle area
```

Comment:
It accepts a pretty big error, so it still works in these cases with the bags I tested with. It more just prevents the extreme case of a giant blob that looks nothing like a circle, like the walls of ground, being the largest thresholded contour.